src/esp/gfx/GenericDrawable.cpp

```diff
@@ -143,7 +160,7 @@ void GenericDrawable::draw(const Mn::Matrix4& transformationMatrix,
               : node_.getSemanticId())
       .setTransformationMatrix(transformationMatrix)
       .setProjectionMatrix(camera.projectionMatrix())
-      .setNormalMatrix(transformationMatrix.normalMatrix());
+      .setNormalMatrix(invTransNormalMat);
```

Comment:
```suggestion
      .setNormalMatrix(normalMatrix);
```

That's what the matrix ultimately is, so I think it's clearest to name it like that. Same for the other drawable.